src/theme/theme.ts

```diff
@@ -40,6 +40,11 @@ export const theme: DefaultTheme = {
       success: colors.peppermintGreen,
       warning: colors.butteryYellow,
       selected: colors.aliceBlue,
+      newDesign: {
```

Comment:
Please change the name to more meaningful

Reply:
changed to 'tracked'